### ISSUE TITLE
Have different default notification level for room vs discussion

### ISSUE THREAD

**Issue by MightyCreak** [vector-im/riot-web#8312]
- **Browser**: Firefox 64.0.2
- **OS**: Fedora, macOS

### Description

By default the notification seems to be set to "All messages". After a small discussion on [#linear-supporters:matrix.org](https://matrix.to/#/!eUGMvloIjhBoAwlyRh:matrix.org/$1548786068122372JJRUF:matrix.org?via=matrix.org&via=kittenface.studio&via=kolosowscy.pl), we thought it would be interesting to have it set to "Mentions only" by default for rooms and "All messages" for discussions.

The rational is that on rooms there are often several people talking together and having a notification each time there's a new line can be upsetting (experienced by my less tech-savvy friends and family who removed Riot notifications altogether either in the apps config or directly on their phone).

My use case is that if I want the attention of someone in a room, I mention him, and my friends tends to do the same as well. But for discussion rooms, I use them as a replacement for my SMS now, so I need a notification each time there's a new message.

### Steps to reproduce

- Create a new room
- See that the default notification level is set to "All messages"

- See that it's also the case for discussion room

**Comment by lampholder** [comment]
By 'discussion room' do you mean 1:1 rooms? (rooms with only two participants)

**Comment by MightyCreak** [comment]
Yep! I call them discussion rooms though because it's not always 1:1, there could be more people in it.

**Comment by lampholder** [comment]
We do now have an item on our roadmap to look into more sensible defaults for notifications! This has been outstanding for a while. I expect we'll do something very similar to the behaviour you describe :)

**Comment by ara4n** [comment]
This is almost a dup of https://github.com/vector-im/riot-web/issues/3268
